CHANGELOG.md

```diff
@@ -5,13 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
 ## Unreleased [3.0.0-alpha]
 
 While this is unreleased, please only add v3 features here.
 Rebasing master onto v3 doesn't require a changelog update.
 
 ### Changed
+
+* Made `CacheHandler` an abstract base class
+
+### Added
+
+* `Scope` - An enum which contains all of the authorization scopes (see [here](https://github.com/plamere/spotipy/issues/652#issuecomment-797461311)).
+
+* Added the following endpoints
+    * `Spotify.current_user_saved_episodes`
+    * `Spotify.current_user_saved_episodes_add`
+    * `Spotify.current_user_saved_episodes_delete`
+    * `Spotify.current_user_saved_episodes_contains`
+    * `Spotify.available_markets`
```

Comment:
Could these be in the v2 still?

Reply:
Yes they can. I was going to suggest that after this PR was merged.

Reply:
Ok! Let's merge this then, thanks!